format/src/main/thrift/schema.thrift

```diff
@@ -187,15 +187,17 @@ struct ParentColumnTableRelation {
 }
 
 struct DataMapSchema  {
+    // DataMap name
+    1: required string dataMapName;
```

Comment:
will the change in sequence affect the previous version?

Reply:
NO, we are adding it in this version only. This `DataMapSchema` does not exist in old version